rllib/env/tests/test_multi_agent_env.py

```diff
@@ -447,7 +448,7 @@ def compute_actions_from_input_dict(
         ev = RolloutWorker(
             env_creator=lambda _: MultiAgentCartPole({"num_agents": 2}),
             default_policy_class=ModelBasedPolicy,
-            config=AlgorithmConfig()
+            config=DQNConfig()
```

Comment:
This test failed after my changes because we are instantiating an instance of DQN here and passing in AlgorithmConfig(), which lacks DQN the defaults that are now not provided anymore.

Reply:
Makes sense. This test is a little strange to begin with anyways :D